### DISCUSSION TITLE
Save Shortcut

### DISCUSSION THREAD

**Discussion by slsimic** [WeblateOrg/weblate#5223] [Ideas] [answered]
Is it only me or doesn't it look logical that when doing some translations and use keyboard shortcut for saving current translation that focus of the cursor should stay in the translation field. This way you always need to reach for the mouse to click back into translation field.

**Comment by nijel** [comment, marked as answer]
That's a bug (and probably not fully deterministic), I've fixed it in 61505187d2.
